config/config.php

```diff
@@ -45,7 +44,6 @@
     // Load development config if it exists
     new PhpFileProvider(realpath(__DIR__) . '/development.config.php'),
 ], $cacheConfig['config_cache_path'], [
-    Laminas\ZendFrameworkBridge\ConfigPostProcessor::class,
```

Comment:
We were not directly depending on this, and now none of our dependencies do either, so I had to remove the line to prevent a runtime error.